app/src/main/java/org/kiwix/kiwixmobile/KiwixMobileActivity.java

```diff
@@ -1166,12 +1166,11 @@ public boolean onKeyDown(int keyCode, KeyEvent event) {
             getCurrentWebView().goBack();
           } else if (isFullscreenOpened) {
             closeFullScreen();
+          } else if (compatCallback.mIsActive) {
```

Comment:
Hey could you directly make it go back to the page of content? I have to press back twice to get back to the page?

Reply:
If implemented, whenever the user will type in something to search and go back to viewing its presence in the content, he/she won't be able to, as the `Find in page` functionality as already been closed by just a single back action. Thus, I think the current flow is correct. Please think it over.

Reply:
Cool. No issues. I didn't think of that